app/src/main/java/dev/animeshvarma/sigil/ui/screens/EncryptionScreen.kt

```
@@ -120,4 +187,310 @@ fun EncryptionInterface(viewModel: SigilViewModel, uiState: UiState) {
+                            }
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+fun ExpandableProfileCard(
```

detekt (1.23.8)

[warning] The function ExpandableProfileCard appears to be too complex based on Cognitive Complexity (complexity: 68). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)